src/components/Table/components/TableRows.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ColumnDefinitionType } from '../Table'
+
+type TableRowsProps<T, K extends keyof T> = {
+    data: Array<T>;
+    columns: Array<ColumnDefinitionType<T, K>>;
+}
+
+const TableRows = <T, K extends keyof T>({data, columns}: TableRowsProps<T, K>): React.ReactElement => {
+    const rows = data.map((row, i) => {
```

Comment:
in this instance i would prefer 'i' and 'j' to be named more explicitly, such as rowIndex, and columnIndex etc, just for readability in the future